src/module/iotjs_module_fs.cpp

```diff
@@ -332,6 +332,23 @@ JHANDLER_FUNCTION(Rmdir) {
 }
 
 
+JHANDLER_FUNCTION(Unlink) {
+  JHANDLER_CHECK(handler.GetThis()->IsObject());
+  JHANDLER_CHECK(handler.GetArgLength() >= 1);
```

Comment:
Although I think that is problem of preference, if you want, I will replace all of the existing usage of >= 1 (e.g., `Close`, `Stat`, `Rmdir`) to > 0 in the iotjs_module_fs.cpp. Please let me know.